racket/collects/racket/contract/private/misc.rkt

```diff
@@ -666,34 +665,21 @@
   (define env (contract-random-generate-get-current-environment))
   (λ () (random-any/c env fuel)))
 
-(define-struct any/c ()
+(define-struct any/c (name)
```

Comment:
Before:
```racket
Welcome to Racket v8.6 [cs].
> (object-name (or/c any/c any/c any/c))
'named-any/c
```

After:
```racket
Welcome to Racket v8.6 [cs].
> (object-name (or/c any/c any/c any/c))
'any/c
```

Reply:
I'm not really sure what's going on here (I can't tell from what I'm reading on github), but I see a diff where a `name` field was added to `any/c` and I don't think we want that. We should stick with using `named-any/c` if we want a name.